applications/Chat/coati/experience_maker/naive.py

```diff
@@ -57,15 +79,30 @@ def make_experience(self, input_ids: torch.Tensor, **generate_kwargs) -> Experie
 
         actor_output = self.actor(sequences, attention_mask)["logits"]
         action_log_probs = calc_action_log_probs(actor_output, sequences, num_actions)
+
         base_model_output = self.initial_model(sequences, attention_mask)["logits"]
+
         base_action_log_probs = calc_action_log_probs(base_model_output, sequences, num_actions)
         value = self.critic(sequences, attention_mask)
-        r = self.reward_model(sequences, attention_mask)
-        reward = compute_reward(r, self.kl_coef, action_log_probs, base_action_log_probs, action_mask=action_mask)
+        sequences_text = self.tokenizer.batch_decode(sequences, skip_special_tokens=True)
+
+        sequences_rm = self.rm_model_tokenizer(
+            sequences_text, return_tensors="pt", padding="max_length", truncation=True, max_length=300
+        )
+        r = self.reward_model(
+            **{
```

Comment:
This is strange. Why not pass kwargs directly? Like `sequences=xxx, attention_mask=xxx`